pranama-ui/src/components/View/services/email_service/ContactPage.vue

```diff
@@ -50,15 +50,8 @@
 </template>
 
 <script>
-import axios from 'axios'
-const apiClient = axios.create({
-    baseURL: 'http://malabarhangouts.com',
-    withCredentials: false,
-    headers: {
-        Accept: 'application/json',
-        'Content-Type': 'application/json'
-    }
-})
+
+import EmailService from "@/services/malabar/EmailService";
```

Comment:
@escuelatech  this file is using the malabar base url https://github.com/escuelatech/Pranama/pull/5#discussion_r416247750